ansible/playbooks/tasks/detect-cloud-platform.yaml

```diff
@@ -16,7 +16,7 @@
       changed_when: false
       failed_when: false
       register: probe_aws
-
+      when: cloud_platform_name is undefined or cloud_platform_name | length == 0
```

Comment:
Like to say: Only attempt autodetection calling the `system-manufacturer` variant of `dmidecode`  if `cloud_platform_name` is not defined by the inventory.yaml